README.md

```diff
@@ -78,9 +78,27 @@ When a change is committed, a Jenkins job will recompile and re-deploy mattermod
 
 In order to test Mattermod locally a couple of steps are needed.
 
-* 1\. Local running version of MYSQL database. You can easily spin up a database by executing ```docker run --name mattermod-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=mattermod -e MYSQL_USER=mattermod -e MYSQL_PASSWORD=mattermod -e MYSQL_DATABASE=mattermod -d mysql:5.7 > /dev/null;```
-* 2\. Download the latest [`Mattermod`](https://github.com/mattermost/mattermost-mattermod) version and update the [`config/config-mattermod.json`](https://github.com/mattermost/mattermost-mattermod/blob/master/config/config-mattermod.json).
-  * 2.1\. You will need to create a Github repo and a Github Access Token. *GithubAccessToken*, *Username* and *Repositories* should be updated with your details.
-  * 2.2\. *DataSource* should be updated with the DSN of the MYSQL Database that was created in step 1.
-  * 2.3\. *PrLabels* should be updated with your preferred PR Labels
-  * 2.4\. The rest of the config file should be updated based on the testing activity. For example if you want to test the spin up of a test server, then all *SetupSpinmintTag*, *SetupSpinmintMessage*, *SetupSpinmintDoneMessage*, *SetupSpinmintFailedMessage*, *DestroyedSpinmintMessage*, *DestroyedExpirationSpinmintMessage* should be updated, as well as *JenkinsCredentials* and *AWSCredentials*.
+1. Since Mattermod needs to consume webhooks from GitHub, we need to expose our webserver to the public internet. It is recommended to use [ngrok](https://ngrok.com/) for this, but you are free to use anything else.
+
+    For Mattermost staff, please visit https://dashboard.ngrok.com/endpoints/domains and reserve a domain under your name, under the correct region. As a convention, please use `<githubusername>.<region>.ngrok.io`.
+    
+    Download ngrok locally and setup your authtoken from this page: https://dashboard.ngrok.com/get-started/setup.
+    
+    Fire up ngrok with `./ngrok http -subdomain=<githubusername> -region=<region> 8086`.
+
+2. Go to https://github.com/mattermosttest/mattermost-server/settings/hooks and add your webhook. The payload URL will point to your ngrok URL. For now the path should be at "/pr_event", but it is going to be split into multiple paths later.
```

Comment:
I dont have access to `https://github.com/mattermosttest/mattermost-server/settings/hooks`

How about adding an note on who or where to ping for settings access?

Reply:
Hmm .. doesn't org admins translate to repo admins? I would have thought so. Let me add you guys.